electron/index.ts

```diff
@@ -182,9 +182,13 @@ app.setAsDefaultProtocolClient('ss');
 
 function interceptShadowsocksLink(argv: string[]) {
   if (argv.length > 1) {
-    const url = argv[1];
-    if (url.startsWith('ss://')) {
+    const protocol = 'ss://';
+    let url = argv[1];
+    if (url.startsWith(protocol)) {
       if (mainWindow) {
+        // The system adds a trailing slash to the intercepted URL (before the fragment).
+        // Remove it before sending to the UI.
+        url = `${protocol}${url.substr(protocol.length).replace(/\//g, '')}`;
```

Comment:
How about...
```ts
url = `${protocol}${url.substr(protocol.length).replace(/^\//, '')}`;
```
?

Reply:
That wouldn't work because the slash is not at the beginning. For a url like `ss://base64#fragment`, the system passes it as `ss://base64/#fragment`.

Reply:
But it calls `replace` on the truncated URL - this is why tests would be handy :-)

That said, we haven't made it striaghtforward yet to add tests in this folder.